neuro_flow/context.py

```diff
@@ -354,6 +356,8 @@ async def create(
                     local=local_path,
                     full_local_path=calc_full_path(ctx, local_path),
                 )
+        ctx = replace(ctx, _volumes=volumes)
```

Comment:
This makes comment on line 346 obsolete (unable to comment that line directly), please update or remove it.